### PR TITLE
Pub 1995 test endpoint integration

### PR DESCRIPTION
**Pull request by ashwini-mv**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-1995

### Change description

Implemented endpoint integration in front end to clear test data
```
[] Yes
[x] No
```

### CODE REVIEW

File: src/test/end-to-end/shared/testingSupportApi.ts [hunk 1]
```diff
@@ -18,22 +17,87 @@ const createFile = (filePath, fileName) => {
...
+        await superagent
+            .delete(`${testConfig.ACCOUNT_MANAGEMENT_BASE_URL}/testing-support/application/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenDataManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete applications test data , http-status: ${e.response?.status}`);
+    }
+};
+
 export const deleteLocation = async (locationId: string) => {
```
Comment (KianKwa) [inline src/test/end-to-end/shared/testingSupportApi.ts:101]: Is this method still needed now we have the testing support one? Is that used in any other tests?

File: src/test/end-to-end/shared/testingSupportApi.ts [hunk 2]
```diff
@@ -18,22 +17,87 @@ const createFile = (filePath, fileName) => {
...
 
+export const clearTestData = async () => {
+    await clearAllPublicationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllSubscriptionsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllLocationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllAccountsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+    await clearAllMediaApplicationsByTestPrefix(testConfig.TEST_SUITE_PREFIX);
+};
+
+export const clearAllPublicationsByTestPrefix = async (testSuitePrefix: string) => {
```
Comment (KianKwa) [inline src/test/end-to-end/shared/testingSupportApi.ts:41]: The only test clean up method exposed outside here is clearTestData? If so the delete methods below does not need to have the 'export' prefix

File: src/test/end-to-end/shared/testingSupportApi.ts (hunk 1 above)
Comment (ashwini-mv) [reply src/test/end-to-end/shared/testingSupportApi.ts:101]: I will remove this

Comment (ashwini-mv) [reply src/test/end-to-end/shared/testingSupportApi.ts]: Removed deleteLocation, delete artefact and delete subscriptions methods

File: src/test/end-to-end/shared/testingSupportApi.ts (hunk 2 above)
Comment (ashwini-mv) [reply src/test/end-to-end/shared/testingSupportApi.ts]: Removed export

**Review by KianKwa — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**